com.arviceblot.eso-addon-manager.json

```diff
@@ -13,7 +13,12 @@
         "--socket=fallback-x11",
         "--socket=wayland",
         "--device=dri",
-        "--filesystem=host"
+        "--filesystem=~/.steam",
+        "--filesystem=~/.local/share/Steam",
+        "--filesystem=~/.var/app/com.valvesoftware.Steam",
+        "--filesystem=~/Games",
+        "--filesystem=xdg-documents/Elder Scrolls Online",
+        "--filesystem=/run/media/mmcblk0p1/steamapps"
```

Comment:
This should be `/run/media`, mmcblk0p1 is not a fixed block device name.

It might be fixed for steam deck, but you can't know that.

Reply:
Ok, I didn't want to make it too generic for most mount points. I'll update it.